libc/src/stdfix/CMakeLists.txt

```diff
@@ -42,6 +42,34 @@ foreach(suffix IN ITEMS hr r lr hk k lk uhr ur ulr uhk uk ulk)
     DEPENDS
       libc.src.__support.fixed_point.fx_bits
   )
+
+  add_entrypoint_object(
+    countls${suffix}
+    HDRS
+      countls${suffix}.h
+    SRCS
+      countls${suffix}.cpp
+    COMPILE_OPTIONS
+      -O3
+    DEPENDS
+      libc.src.__support.fixed_point.fx_bits
+  )
+
+  if(LIBC_COMPILER_HAS_PADDING_ON_UNSIGNED_FIXED_POINT)
```

Comment:
Why do we need these extra entry points built with this flag?  Are they supposed to be on or off by default?
If it's only for testing with and without the `-Xclang=-fpadding-on-unsigned-fixed-point`, then using `FLAGS` in the main entry points is more apt.

Reply:
> Why do we need these extra entry points built with this flag? Are they supposed to be on or off by default? If it's only for testing with and without the `-Xclang=-fpadding-on-unsigned-fixed-point`, then using `FLAGS` in the main entry points is more apt.

Yes, the idea is to run all countlsfx tests twice. Once with `-Xclang=-fpadding-on-unsigned-fixed-point` and once without. I'm not sure what you mean by "using `FLAGS` in the main entry point". 

Are there any other places in the codebase where we run the same test suite twice but differ the flags? If so, I can mirror what is done there.